pkgs/by-name/ko/komga/package.nix

```diff
@@ -32,8 +32,11 @@ stdenvNoCC.mkDerivation rec {
     description = "Free and open source comics/mangas server";
     homepage = "https://komga.org/";
     license = lib.licenses.mit;
-    platforms = jdk17_headless.meta.platforms;
-    maintainers = with lib.maintainers; [ govanify ];
+    platforms = jdk23_headless.meta.platforms;
+    maintainers = with lib.maintainers; [
+      tebriel
+      govanify
+    ];
     mainProgram = "komga";
   };
```

Comment:
Could also use `sourceProvenance = with lib.sourceTypes; [ binaryBytecode ];`.

Reply:
🤦🏻 yes totally.

Reply:
commits re-ordered and sourceProvenance added to meta field